interbotix_ros_xscobots/interbotix_xscobot_moveit/launch/moveit_api.launch.py

```diff
@@ -0,0 +1,340 @@
+"""
+A launch file for running the motion planning python api tutorial
+"""
+import os
+import sys
+import yaml
+from ament_index_python.packages import get_package_share_directory
+from launch import LaunchDescription
+from launch_ros.actions import Node
+from launch.actions import DeclareLaunchArgument, IncludeLaunchDescription, OpaqueFunction, TimerAction
+from launch.conditions import IfCondition, LaunchConfigurationEquals
+from launch.launch_description_sources import PythonLaunchDescriptionSource
+from launch.substitutions import (
+    LaunchConfiguration,
+    PathJoinSubstitution,
+    PythonExpression,
+    TextSubstitution,
+)
+from launch_ros.actions import Node
+from launch_ros.substitutions import FindPackageShare
+from launch_ros.parameter_descriptions import ParameterValue
+
+from moveit_configs_utils import MoveItConfigsBuilder
+
+from interbotix_xs_modules.xs_launch import (
+    construct_interbotix_xscobot_semantic_robot_description_command,
+    declare_interbotix_xscobot_robot_description_launch_arguments,
+    determine_use_sim_time_param,
+)
+
+from interbotix_xs_modules.xs_common import (
+    get_interbotix_xscobot_models,
+)
+
+def load_yaml(package_name, file_path):
+    package_path = get_package_share_directory(package_name)
+    absolute_file_path = os.path.join(package_path, file_path)
+
+    try:
+        with open(absolute_file_path, 'r') as file:
+            return yaml.safe_load(file)
+    except EnvironmentError:  # parent of IOError, OSError *and* WindowsError where available
+            return None
+
+def launch_setup(context, *args, **kwargs):
+
+    robot_model_launch_arg = LaunchConfiguration('robot_model')
+    robot_name_launch_arg = LaunchConfiguration('robot_name')
+    base_link_frame_launch_arg = LaunchConfiguration('base_link_frame')
+    # show_ar_tag_launch_arg = LaunchConfiguration('show_ar_tag')
```

Comment:
Remove this since we've added it in the `declare_interbotix_xscobot_robot_description_launch_arguments` function.
```suggestion
```

Reply:
Sure